orchestra/model/component.py

```diff
@@ -214,4 +214,5 @@ def yamldump(data):
         default_style="|",
         width=100000,
         sort_keys=True,
+        Dumper=yaml.CDumper,
```

Comment:
Why not `CSafeDumper`?

Reply:
Substituted all usages with `CSafeLoader` and `CSafeDumper`